packages/extensions/openapi-to-cadl/src/utils/decorators.ts

```diff
@@ -72,8 +72,8 @@ export function getPropertyDecorators(property: Property): CadlDecorator[] {
 
   if (property.serializedName !== property.language.default.name) {
     decorators.push({
-      name: "friendlyName",
-      arguments: [property.language.default.name],
+      name: "projectedName",
+      arguments: ["json", property.serializedName],
```

Comment:
Should `json` be hardcoded here? Or could there be cases where it's `xml` for instance? I think that info probably shows up somewhere in the tree

Reply:
I think cadl is lacking full xml support for now so I'm fine keeping it like that until we want to convert an xml service and get the bigger picture